public/index.html

```diff
@@ -5,16 +5,15 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
**suggestion(folder)**: We need not move the code into the public folder. The public folder should be separated from the actual code. I Will let you know about the process for setting this up in the upcoming meeting.